Source/Krypton Toolkit/Krypton Toolkit Suite Extended 2019.sln

```diff
@@ -263,7 +262,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Krypton.Toolkit.Suite.Exten
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CheckSum", "Examples\CheckSum\CheckSum.csproj", "{022DBF8D-4DCB-41BA-A7EB-BF40B46C4622}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Krypton.Toolkit.Suite.Extended.Memory.Box 2019", "Krypton.Toolkit.Suite.Extended.Memory.Box\Krypton.Toolkit.Suite.Extended.Memory.Box 2019.csproj", "{CB14B073-F49D-4EBA-B823-B6CF6887F37E}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Krypton.Toolkit.Suite.Extended.Memory.Box 2019", "Krypton.Toolkit.Suite.Extended.Memory.Box\Krypton.Toolkit.Suite.Extended.Memory.Box 2019.csproj", "{CB14B073-F49D-4EBA-B823-B6CF6887F37E}"
```

Comment:
Why does this one have `2019` on it

Reply:
Following the naming convention of the standard toolkit. Will be changed to 2022, once VS 2022 is GA.

Reply:
But none of the other Extended ones have it set (Yet!)

Reply:
That's odd, as they're all labelled so in the build I'm working on

![image](https://user-images.githubusercontent.com/949607/125940958-e20b5753-2471-4d79-b76e-dfb0ba8bb579.png)

![image](https://user-images.githubusercontent.com/949607/125941039-bf48b6c8-9c11-40a7-8f5b-3de414aff9b4.png)